ios/demo/src/ViewController.h

```diff
@@ -0,0 +1,15 @@
+//
+//  ViewController.h
+//
+//  Created by Karim Naaji on 10/12/16.
+//  Copyright © 2016 Karim Naaji. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import "TGMapViewController.h"
+
+@interface ViewController : TGMapViewController
```

Comment:
You should name this something other than "ViewController" as that's *really* close to apple's stuff. Maybe MapViewController